rules/DowngradePhp81/Rector/FuncCall/DowngradeArrayIsListRector.php

```diff
@@ -97,16 +95,15 @@ function (Node $subNode): bool {
                 }
 
                 // need pull Scope from target traversed sub Node
-                $scope = $subNode->getAttribute(AttributeKey::SCOPE);
-                return ! $this->shouldSkip($subNode, $scope);
+                return ! $this->shouldSkip($subNode);
             }
         );
 
         if (! $expr instanceof FuncCall) {
             return null;
         }
 
-        $variable = new Variable($this->variableNaming->createCountedValueName('arrayIsList', $scope));
+        $variable = new Variable('arrayIsListFunction');
```

Comment:
this will make error if `arrayIsListFunction` variable exists, the `VariableNaming` here was on purpose to avoid variable replace existing variable.